src/Compilers/VisualBasic/Test/Semantic/IOperation/IOperationTests_IVariableDeclaration.vb

```diff
@@ -0,0 +1,346 @@
+    Initializer: IObjectCreationExpression (Constructor: Sub Program.C..ctor()) (OperationKind.ObjectCreationExpression, Type: Program.C)
+]]>.Value
+
+            VerifyOperationTreeForTest(Of LocalDeclarationStatementSyntax)(source, expectedOperationTree)
+        End Sub
+
+        <Fact(), WorkItem(17599, "https://github.com/dotnet/roslyn/issues/17599")>
+        Public Sub MultipleDimAsNew()
+            Dim source = <![CDATA[
+Imports System
+Imports System.Collections.Generic
+Imports System.Linq
+
+Module Program
+    Sub Main(args As String())
+        Dim i1, i2 As New Integer'BIND:"Dim i1, i2 As New Integer"
+    End Sub
+End Module
+    ]]>.Value
+
+            Dim expectedOperationTree = <![CDATA[
+IVariableDeclarationStatement (2 variables) (OperationKind.VariableDeclarationStatement)
+  IVariableDeclaration: i1 As System.Int32 (OperationKind.VariableDeclaration)
+    Initializer: IObjectCreationExpression (Constructor: Sub System.Int32..ctor()) (OperationKind.ObjectCreationExpression, Type: System.Int32)
+  IVariableDeclaration: i2 As System.Int32 (OperationKind.VariableDeclaration)
+    Initializer: IObjectCreationExpression (Constructor: Sub System.Int32..ctor()) (OperationKind.ObjectCreationExpression, Type: System.Int32)
+]]>.Value
+
+            VerifyOperationTreeForTest(Of LocalDeclarationStatementSyntax)(source, expectedOperationTree)
+        End Sub
+
+
+
+        <Fact(), WorkItem(17599, "https://github.com/dotnet/roslyn/issues/17599")>
+        Public Sub MixedDimAsNewAndEqualsDeclarations()
+            Dim source = <![CDATA[
+Imports System
+Imports System.Collections.Generic
+Imports System.Linq
+
+Module Program
+    Sub Main(args As String())
+        Dim i1, i2 As New Integer, b1 As Boolean = False'BIND:"Dim i1, i2 As New Integer, b1 As Boolean = False"
+    End Sub
+End Module
+    ]]>.Value
+
+            Dim expectedOperationTree = <![CDATA[
+IVariableDeclarationStatement (3 variables) (OperationKind.VariableDeclarationStatement)
+  IVariableDeclaration: i1 As System.Int32 (OperationKind.VariableDeclaration)
+    Initializer: IObjectCreationExpression (Constructor: Sub System.Int32..ctor()) (OperationKind.ObjectCreationExpression, Type: System.Int32)
+  IVariableDeclaration: i2 As System.Int32 (OperationKind.VariableDeclaration)
+    Initializer: IObjectCreationExpression (Constructor: Sub System.Int32..ctor()) (OperationKind.ObjectCreationExpression, Type: System.Int32)
+  IVariableDeclaration: b1 As System.Boolean (OperationKind.VariableDeclaration)
+    Initializer: ILiteralExpression (Text: False) (OperationKind.LiteralExpression, Type: System.Boolean, Constant: False)
+]]>.Value
+
+            VerifyOperationTreeForTest(Of LocalDeclarationStatementSyntax)(source, expectedOperationTree)
+        End Sub
+
+
+
+        <Fact(Skip:="https://github.com/dotnet/roslyn/issues/17913"), WorkItem(17599, "https://github.com/dotnet/roslyn/issues/17599")>
```

Comment:
It is not clear that https://github.com/dotnet/roslyn/issues/17913 is a product issue. What is the reason to skip this?